README.md

```diff
@@ -5,37 +5,54 @@ This implementation assumes that you have at least Asterisk 12.0.0 running. This
+    {                                                                              
+      "42": {                                                             
+        "trunks": [                                                                
+          "42-A"
+        ]                                                                      
+        "stations": [                                                        
+          "SIP/phone1",
+          "SIP/phone2"                                            
+        ]                                                                
+      }                                                                            
+    }                                                                              
   ]                                                                              
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Right now, however, this application only supports one trunk per sharedExtension, and the sharedExtension MUST have the same name as the trunk. The support for multiple trunks (such as 42A and 42B) will come later.
+Right now, however, this application only supports one trunk per sharedExtension, it can have any name, however, but the SIP trunk must appear in sip.conf, like below: 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+[42-A]
+type = peer
+host = 0.0.0.0
+username = 42-A
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You must also have a dialplan extension in extensions.conf that leads to the application (must have same name as what application is being started in the code) and that has an argument to represent the SLA bridge to reach.  An example is below:
+You must also have a dialplan extension in extensions.conf that leads to the application (must have same name as what application is being started in the code) and that has an argument to represent the SLA bridge to reach.  An example with a device state hint is below:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+exten => 42,hint,Stasis:42
 exten => 42,1,NoOp()                                                             
     same => n,Stasis(sla,42)                                                    
     same => n,Hangup()
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If an inbound caller were to be kicked out of this shared extension, they would be hungup. You could also make the dialplan call the application again to redirect a user to another shared extension like below:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+exten => 42,hint,Stasis:42
+exten => 42,1,NoOp()                                                             
+    same => n,Stasis(sla,42)
+    same => n,Goto(420,1)                                                 
+
+exten => 420,hint,Stasis:420
+exten => 420,1,NoOp()
+    same => n,Stasis(sla,420)
+    same => n,Hangup()
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ARI patch for detecting channel hold events is also required (may be in the Asterisk 13 branch by the time we are finished). 
```

Comment:
Remove this sentence.

Instead, note that the minimum version of Asterisk required for this ARI application is 13.4.0.